pandas/tseries/offsets.py

```diff
@@ -1090,12 +1090,15 @@ def apply(self, other):
 
 
 class CustomBusinessMonthEnd(_CustomBusinessMonth):
-    __doc__ = _CustomBusinessMonth.__doc__.replace('[BEGIN/END]', 'end')
+    if _CustomBusinessMonth.__doc__:
+        __doc__ = _CustomBusinessMonth.__doc__.replace('[BEGIN/END]', 'end')
     _prefix = 'CBM'
 
 
 class CustomBusinessMonthBegin(_CustomBusinessMonth):
```

Comment:
I think better to do this with ``@subsitution`` and template this. We dont' use this pattern anywhere and might be confusing.

Reply:
Makes sense - I'll take a look and repush